### DISCUSSION TITLE
[@types/chrome] chrome.tabGroups.ColorEnum "orange" is not present

### DISCUSSION THREAD

**Discussion by jkeegan2** [DefinitelyTyped/DefinitelyTyped#60311] [Issues with a @types package]
Orange is not present in the chrome.tabGroups.ColorEnum, but it should be according to online documentation here: https://developer.chrome.com/docs/extensions/reference/tabGroups/#type-Color

[a comment by an automated account is not shown]

**Comment by MysteryBlokHed** [comment]
I just opened #60317
